docs/documentation/server_admin/topics/organizations/authenticating-members.adoc

```diff
@@ -0,0 +1,99 @@
+[id="authenticating-members_{context}"]
+
+=  Authenticating members
+[role="_abstract"]
+One of the main changes introduced by organizations when enabled to a realm is on how users authenticate.
+These changes are mainly about being able to recognize if a user is authenticating in the context of an organization
+and changing the authentication flow accordingly on a per-organization basis.
```

Comment:
```suggestion
When you enable organizations for a realm, user authentication is changed. A user is authenticated in the context of an organization, which changes the authentication flow on a per-organization basis.
```

Reply:
The user is not always authenticating in the context of the organization as it depends on `recognizing` if the user actually maps to an organization.

That is what I tried to make more explicit there.

Reply:
Perfect :)